spec/takeaway_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'takeaway'
+require 'menu'
+
+describe Takeaway do
+
+  let(:menu) { Menu.new }
```

Comment:
I might be wrong with this, but in order to separate your unit tests via class, you would probably want to read in a double here for the menu class rather than initializing a new class in your takeaway tests.